.github/workflows/frontend-ci.yml

```
@@ -0,0 +1,70 @@
+
+      - name: Build Storybook
+        run: npm run sb:build
+
+      - name: Upload Storybook artifact
+        if: always()
+        uses: actions/upload-artifact@v4
+        with:
+          name: storybook-static
+          path: frontend/storybook-static   # IMPORTANT: path is relative to the repository root
+
+      - name: Upload coverage to Codecov
```

actionlint (1.7.7)

step must run script with "run" section or run action with "uses" section

(syntax-check)